test/core/transport/chttp2/too_many_pings_test.cc

```diff
@@ -383,11 +395,11 @@ TEST_F(KeepaliveThrottlingTest, KeepaliveThrottlingMultipleChannels) {
       expected_keepalive_time_sec);
   EXPECT_EQ(PerformWaitingCall(channel, server, cq),
             GRPC_STATUS_DEADLINE_EXCEEDED);
-  // Since the subchannel is shared, the second channel should also have
-  // keepalive settings in sync with the server.
+  // Since the subchannel is not shared any more, the second channel should not
```

Comment:
Please revert this change. After https://github.com/grpc/grpc/pull/28793 the subchannels should be shared.

Reply:
Thanks for pointing it out!